ports/rp2/Makefile

```diff
@@ -28,7 +30,9 @@ else
 BUILD ?= build-$(BOARD)
 endif
 
-$(VERBOSE)MAKESILENT = -s
+ifeq ($(BUILD_VERBOSE),1)
+    MAKE_ARGS += VERBOSE=1  # Picked up in Makefile generated by CMake
```

Comment:
Maybe un-indent this, to match the ifdef's below?

Reply:
Oops, yes thanks. Extra indenting in Makefiles is a dangerous game :laughing: